contracts/Trustcoin.sol

```diff
@@ -72,14 +72,27 @@ contract Trustcoin is OutgoingMigrationTokenInterface, ERC20TokenInterface, Safe
 
   // See ERC20
   function approve(address _spender, uint256 _value) external returns (bool) {
-    allowed[msg.sender][_spender] = _value;
-    Approval(msg.sender, _spender, _value);
+    // https://github.com/ethereum/EIPs/issues/20#issuecomment-263524729
+    int256 valueToSet = 0;
+    if (
+      allowed[msg.sender][_spender] < 0 &&
+      ((_value != 0) && (allowed[msg.sender][_spender] + int(block.number)) < 0)
```

Comment:
I guess it can't, what do you think the const `MIN_BLOCK_DELTA` should be? Or is there even any point in this check at all?